apps/mobile/android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.6-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.5-bin.zip
```

Comment:
What's the difference here?

Reply:
It's a minor version lower for the build tool gradle, but i have been having a ton of issues with `8-6`. To fix it I deleted `./gradle` and allowed Android Studio to reinstall / rebuild gradle dependencies and it aut-generated this version. Since nobody here really cares / uses gradle other than implicitly through `pnpm run android` and when building I don't think this minor version change matters to us at all.